services/metadata/conf/main/logback.xml

```diff
@@ -11,7 +11,7 @@
   </appender>
 
 
-  <Logger name="donkey" level="debug" />
+  <Logger name="terrain" level="debug" />
```

Comment:
If you want to write it up and put it into `logging/metrics` category, I'll take care of it later